taskcluster/mozillavpn_taskgraph/parameters.py

```diff
@@ -12,13 +12,26 @@ def get_defaults(repo_root):
     return {
         "pull_request_number": None,
         "version": "",
+        "app_version":"0.0.0"
     }
 
+def getAppVersion():
+    f = open("CMakeLists.txt","r")
+    lines = f.readlines()
+    f.close()
+    isVersionLine = lambda x: "project(\"Mozilla VPN\" VERSION" in x
+    version_line = next(filter(isVersionLine,lines))
+    tokens = version_line.split(" ")
+    version = tokens[tokens.index("VERSION")+1]
+    return version
+
 
 extend_parameters_schema(
     {
         Required("pull_request_number"): Any(All(int, Range(min=1)), None),
         Required("version"): str,
+        Required("app_version"): str,
```

Comment:
Ah ok, so these indexes aren't for releases then, but rather builds. We may want to rethink the naming a bit in that case.

Instead of introducing a new `app_version` parameter, could we change `version` to read from the `CMakeLists` file instead?